CMakeLists.txt

```diff
@@ -103,21 +101,33 @@ if (${IAS_IS_HOST_BUILD})
   include(CMakeLists.avb_tests.txt)
 endif()
 
+#------------------------------------------------------------------
+#Install
+#------------------------------------------------------------------
+
+install(CODE "execute_process(COMMAND cp ${CMAKE_CURRENT_SOURCE_DIR}/deps/gptp/linux/build/obj/daemon_cl ${CMAKE_INSTALL_PREFIX}/bin/daemon_cl)")
+install(CODE "execute_process(COMMAND mkdir -p /etc/alsa/conf.d)")
```

Comment:
Why not use install(FILES) or install(PROGRAMS)? Then you don't have to run the commands, cmake actually tracks the installed files, etc. See https://cmake.org/cmake/help/v3.0/command/install.html